app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -181,6 +186,10 @@ protected void onPostExecute(Boolean result) {
                 .setProgress(0,0,false)
                 .setOngoing(false)
                 .setPriority(PRIORITY_HIGH);
+        String urlForDelete=baseUrl+media.getFilename();
```

Comment:
spacing before and after `=` and `+`

Reply:
Sorry for that repeated mistake. I'll take care of it from next time